app/utils/auth.js

```diff
@@ -0,0 +1,40 @@
+const AUTH_SECRET = 'auth-secret';
```

Comment:
I haven't used `auth0-lock` yet, quick question: You're calling `checkSecret` to check `AUTH_SECRET` string in the callback handler,  Is `AUTH_SECRET` a public key? I guess the const can be accessed on the client so is it 100% safe and the right way to do this?